doc/lsst.ip.diffim/tasks/lsst.ip.diffim.DecorrelateALKernalSpatialTask.rst

```diff
@@ -0,0 +1,28 @@
+.. lsst-task-topic:: lsst.ip.diffim.DecorrelateALKernelSpatialTask
+
+##########################
+DecorrelateALKernelSpatialTask
+##########################
+
+Pipe-task that removes the neighboring-pixel covariance in an
+image difference that are added when the template image is
+convolved with the Alard-Lupton PSF matching kernel.
+
+.. _lsst.ip.diffim.DecorrelateALKernelSpatialTask-description:
+
+Description
+==================
+
+This task is a simple wrapper around @ref DecorrelateALKernelTask,
+which takes a `spatiallyVarying` parameter in its `run` method. If
+it is `False`, then it simply calls the `run` method of @ref
+DecorrelateALKernelTask. If it is True, then it uses the @ref
+ImageMapReduceTask framework to break the exposures into
+subExposures on a grid, and performs the `run` method of @ref
+DecorrelateALKernelTask on each subExposure. This enables it to
+account for spatially-varying PSFs and noise in the exposures when
+performing the decorrelation.
+
+This task has no standalone example, however it is applied as a
+subtask of pipe.tasks.imageDifference.ImageDifferenceTask.
```

Comment:
These should be a link: `~pipe.tasks.imageDifference.ImageDifferenceTask`

Note: the tilde makes the link text only show the object name, which is what we typically want: https://developer.lsst.io/restructuredtext/style.html#customizing-the-link-text